src/liballoc/arc.rs

```diff
@@ -1211,8 +1360,49 @@ impl<T> From<T> for Arc<T> {
     }
 }
 
+#[unstable(feature = "shared_from_slice", issue = "40475")]
```

Comment:
Right now we unfortunately don't have stability checking on impl blocks, so you can go ahead and tag these impls as `#[stable]`